qiskit_nature/second_q/mappers/parity_mapper.py

```diff
@@ -54,5 +78,53 @@ def pauli_table(cls, nmodes: int) -> list[tuple[Pauli, Pauli]]:
 
         return pauli_table
 
+    def convert(self, operator: SparsePauliOp) -> SparsePauliOp:
```

Comment:
Does this need to be a public method? If so, what is the reasoning for the naming? Would a more descriptive name like `two_qubit_reduce()` be better?